### DISCUSSION TITLE
chatonly option for chat embed option

### DISCUSSION THREAD

**Discussion by martyjoy** [owncast/owncast#3505] [Ideas]
Hi,

  Would it be possible to have a 3rd embed chat option called chatonly?  We currently have readonly and readwrite; however, neither of these options provides us a way to grab the chat window that does not get cluttered with system messages and name changes (both of which are great to have visible to the normal users).  Which is why I'd propose a chatonly option that would only show actual chat messages (the same list we can filter from the admin page) and exclude the name changes and stream name changes, etc.

  Thanks in advance for your time and effort in this project, would be happy to help any odd way I can.

- Marty

**Comment by gabek** [comment]
So here's the thing with these kinds of requests. It's a perfectly fine request. However, you want a view of chat with only chat messages. But maybe the next person wants one with only join messages. And the next person wants one with only name changes. And another wants one with only system messages. And it goes on and on.

I would recommend using CSS to create the type of view you want. You can hide any type of message or component and customize it to your liking without Owncast having to change anything internally.

There's some starting points that might be helpful here: https://owncast.online/docs/appearance/
